auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -469,25 +546,265 @@ public Builder(@SupportedProvider @NonNull String providerId) {
+
+            /**
+             * @param number the phone number in international format
+             * @see #setDefaultNumber(String, String)
+             */
             @NonNull
+            public PhoneBuilder setDefaultNumber(@NonNull String number) {
+                if (getParams().containsKey(ExtraConstants.EXTRA_COUNTRY_ISO)
+                        || getParams().containsKey(ExtraConstants.EXTRA_NATIONAL_NUMBER)) {
+                    throw new IllegalStateException("Cannot overwrite previously set phone number");
+                }
+
+                getParams().putString(ExtraConstants.EXTRA_PHONE, number);
+
+                return this;
+            }
+
+            /**
+             * Set the default phone number that will be used to populate the phone verification
+             * sign-in flow.
+             *
+             * @param iso    the phone number's country code
+             * @param number the phone number in local format
+             */
+            @NonNull
+            public PhoneBuilder setDefaultNumber(@NonNull String iso, @NonNull String number) {
+                if (getParams().containsKey(ExtraConstants.EXTRA_PHONE)) {
+                    throw new IllegalStateException("Cannot overwrite previously set phone number");
+                }
+
+                getParams().putString(ExtraConstants.EXTRA_COUNTRY_ISO, iso);
+                getParams().putString(ExtraConstants.EXTRA_NATIONAL_NUMBER, number);
+
+                return this;
+            }
+
+            /**
+             * Set the default country code that will be used in the phone verification sign-in
+             * flow.
+             *
+             * @param iso country iso
+             */
+            @NonNull
+            public PhoneBuilder setDefaultCountryIso(@NonNull String iso) {
+                if (getParams().containsKey(ExtraConstants.EXTRA_PHONE)
+                        || getParams().containsKey(ExtraConstants.EXTRA_COUNTRY_ISO)
+                        || getParams().containsKey(ExtraConstants.EXTRA_NATIONAL_NUMBER)) {
+                    throw new IllegalStateException("Cannot overwrite previously set phone number");
+                }
+
+                getParams().putString(ExtraConstants.EXTRA_COUNTRY_ISO, iso);
+
+                return this;
+            }
+        }
+
+        /**
+         * {@link IdpConfig} builder for the Google provider.
+         */
+        public static final class GoogleBuilder extends Builder {
+            public GoogleBuilder() {
+                //noinspection deprecation taking a hit for the backcompat team
+                super(GoogleAuthProvider.PROVIDER_ID);
+                if (getApplicationContext().getString(R.string.default_web_client_id)
+                        .equals(UNCONFIGURED_CONFIG_VALUE)) {
+                    throw new RuntimeException("Check your google-services plugin configuration," +
+                            " the client id string wasn't populated.");
```

Comment:
nit: error message should specifically mention `default_web_client_id`)

Reply:
Done.